relay-dynamic-config/src/defaults.rs

```diff
@@ -395,5 +400,51 @@ fn span_metrics(is_extract_all: bool) -> impl IntoIterator<Item = MetricSpec> {
                     .always(), // already guarded by condition on metric
             ],
         },
+        MetricSpec {
+            category: DataCategory::Span,
+            mri: "d:spans/webvital.score.total@ratio".into(),
```

Comment:
Please add the new MRIs to https://github.com/getsentry/sentry/blob/master/src/sentry/sentry_metrics/indexer/strings.py before merging this.

Reply:
https://github.com/getsentry/sentry/pull/63699